src/cbmc_starter_kit/setup.py

```diff
@@ -73,8 +73,12 @@ def main():
     source_root = repository.repository_root()
     if shutil.which("litani"):
         litani = Path("litani")
-    else:
+    elif repository.litani_root() is not None:
         litani = repository.litani_root() / "litani"
+    else:
+        print("Error: could not find litani root. Is litani installed?")
```

Comment:
I'm not sure if this should be a print or a `logging.error`.

Reply:
I think `logging.error` would make more sense actually, will add that in tomorrow.